.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -21,4 +21,6 @@ Closes # <-- _insert Issue number if one exists_
 - [ ] performed checkstyle check locally?
 - [ ] added/updated copyright headers?
 - [ ] documented public classes/methods?
-- [ ] added/updated relevant documentation?
\ No newline at end of file
+- [ ] added/updated relevant documentation?
+- [ ] added relevant details to the changelog?
+- [ ] linked GitHub project?
```

Comment:
Assuming the intention was to link the GH project that _uses_ the EDC, I'd clearly state that:
```suggestion
- [ ] linked GitHub project where EDC is used?
```

Reply:
No, I meant the project tab on the right-hand side:
![image](https://user-images.githubusercontent.com/72392527/156578398-df5fe13f-71d9-4835-a100-74a36342aa4f.png)
As this is restricted to the ones we have in this repo and the EDC GitHub org, this has automatically an EDC reference.